src/pages/Calendar/Calendar.js

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from 'react';
+import { CalendarLibrary } from './Components/CalendarLibrary';
+import CalendarCheckBox from './Components/CalendarCheckBox';
+import CalendarModal from './Components/CalendarModal';
+import useFetch from '../../hooks/useFetch';
+import CalendarStatusModal from './Components/CalendarStatusModal';
+import * as S from './Calendar.styles';
+
+const testUrl = '/data/testAll.json';
+const userUrl = '/data/userCalendar.json';
+
+export default function Calendar() {
+  const [createBtn, setCreateBtn] = useState(false);
+  const [deleteID, setDeleteID] = useState('');
+  const [calendarToggle, setCalendarToggle] = useState({
+    calpick: false,
+    myCalendar: false,
+  });
+  const [calendarCreateInfo, setCalendarCreatreInfo] = useState({
+    title: '',
+    start: '',
+    end: '',
+    comments: '',
+  });
+
+  const [testData, ,] = useFetch(testUrl);
+  const [userData, ,] = useFetch(userUrl);
+
+  const InputValidation =
+    calendarCreateInfo.title &&
+    calendarCreateInfo.start &&
+    calendarCreateInfo.end &&
+    calendarCreateInfo.comments &&
+    calendarCreateInfo.start < calendarCreateInfo.end;
+
+  const handleCreate = () => {
+    setCreateBtn(!createBtn);
+  };
```

Comment:
```suggestion
  const handleCreate = () => {
    setCreateBtn(prev => !prev);
  };
```

state의 이전 값을 가져와 변경 혹은 추가할 때는 함수형 업데이트를 적용하는 것이 안전한 방법입니다.